graalpy/graalpy-openai-starter/src/main/java/com/example/App.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at https://opensource.org/license/UPL.
+ */
+
+package com.example;
+
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.PolyglotException;
+import org.graalvm.python.embedding.utils.GraalPyResources;
+
+import java.util.List;
+
+public class App {
+
+    public static void main(String[] args)  {
+        try (Context context = GraalPyResources.createContext()) {
+            ChatCompletion chatCompletion = context.eval("python",
+                    // language=python
+                    """
+                            import os
+                            from openai import OpenAI
+                            
+                            client = OpenAI(
+                                # This is the default and can be omitted
+                                api_key=os.environ.get("OPENAI_API_KEY"),
+                            )
+                            
+                            chat_completion = client.chat.completions.create(
+                                messages=[
+                                    {
+                                        "role": "user",
+                                        "content": "Say this is a test",
```

Comment:
Wouldn't it be slightly more exciting to insert the args here (or just arg[0])?

Reply:
That requires us to deviate from the original usage example...I think I found an easy way to do this.